frontend/test/lit/test_instrumentation_console.py

```diff
@@ -15,12 +15,15 @@
 # Redirect stderr to stdout to make the output visible to FileCheck.
 # RUN: %PYTHON %s 2>&1 | FileCheck %s
 
+import jax
 import numpy as np
 import pennylane as qml
 
 from catalyst import qjit
 from catalyst.debug import instrumentation
 
+jax.config.update("jax_platforms", "cpu")
```

Comment:
With pytest, global updates like this can affect other tests as well. I don't know how it is with lit, but we may want to unset the config again at the end of the test just to be safe.

Reply:
Done